### DISCUSSION TITLE
[Forms] Feature request: Dropdown field should be able to enforce a selection with "validation: true"

### DISCUSSION THREAD

**Discussion by bmunkholm** [community/community#42187] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

The "validation: true" is meaningless for dropdown fields as one selection is already made by default.
It would be very useful to be able to ensure people select one value in the dropdown different from the currently selected first option.
The first option could be the placeholder text like "Select", "Choose", "" or whatever people would want - perhaps grayed out. When "validation: true" people would be forced to select something else than the initial value.

**Comment by rarkins** [comment]
I agree that this would be very useful. Otherwise we need to add a dummy first option of our own, but then we can't validate.

**Comment by mistic100** [comment]
This should be adressed, currently the `validation: true` has no effect on dropdowns (beside the red asterisk).
